src/containers/org/components/RemoveMemberFromOrgModal.js

```diff
@@ -69,22 +66,14 @@ const RemoveMemberFromOrgModal = ({
     );
   };
 
-  const handleOnClose = () => {
-    onClose();
-    // the timeout avoids rendering the modal with new state before the transition animation finishes
-    setTimeout(() => {
-      dispatch(resetRequestState([REMOVE_MEMBER_FROM_ORGANIZATION]));
-    }, 1000);
-  };
-
   return (
     <ActionModal
         isVisible={isVisible}
         onClickPrimary={handleOnClickPrimary}
-        onClose={handleOnClose}
+        onClose={onClose}
         requestState={removeMemberRS}
         requestStateComponents={rsComponents}
-        textTitle="Confirm Removal" />
+        textTitle="Delete Person" />
```

Comment:
super nitpicky but i feel like "Remove Person" is slightly more clear... "Delete Person" sounds like you're deleting the person altogether. but like i said, very nitpicky—i bet the UI context makes it clear what's happening

Reply:
I agree. same goes for the language surrounding add/deleting a role for a person when really it's assigning and unassigning